tests/testthat/line_breaks_fun_call/line_breaks_and_comments-in.R

```diff
@@ -0,0 +1,15 @@
+call( # comment
+  am
+)
+
+call(
+  # comment
+  am
+)
+
+call(am # comment
+) # FIXME: maybe move ')' up and switch position with the comment
```

Comment:
Better push `am` to its own line then.

Reply:
Ok. So that means when checking whether an expression is multi-line, we also consider the closing `)`. So in general,
```r
call(a
)
```
Becomes
```r
call(
  a
)
```
Instead of 
```r
call(a)
```

Reply:
I'd say only if a comment is involved, but both solutions work.

Reply:
Ok, I prefer the former then.